browser/ui/tabs/brave_tab_menu_model.h

```diff
@@ -87,6 +89,8 @@ class BraveTabMenuModel : public TabMenuModel {
       const std::vector<int>& indices);
 #endif  // BUILDFLAG(ENABLE_CONTAINERS)
 
+  void BuildItemForCustomization(TabStripModel* tab_strip_model, int tab_index);
```

Comment:
Consider adding a comment explaining the role of this function because I can't quite surmise that by the name alone, and I can see an insertion is happening inside it, but I'm not sure exactly why.